public/views/sellCoinbase.html

```diff
@@ -0,0 +1,179 @@
+            ng-attr-placeholder="{{'Choose your source wallet'}}" 
+            ng-model="sell.selectedWalletName" required>
+            <a class="postfix size-12 m0 text-gray">
+              <i class="icon-wallet size-18"></i>
+            </a>
+          </div>
+        </div>
+
+        <label>
+          Amount 
+          <span 
+            ng-if="index.coinbaseToken"
+            ng-init="sell.getPrice(index.coinbaseToken)"
+            ng-show="sell.sellPrice" 
+            class="size-11 text-light right">
+            1 BTC <i class="icon-arrow-right"></i> {{sell.sellPrice.amount}} {{sell.sellPrice.currency}}
+          </span>
+        </label>
+        <div class="input">
+          <input ng-show="!showAlternative" type="number" id="amount" 
+          name="amount" ng-attr-placeholder="{{'Amount in ' + (showAlternative ? 'USD' : 'BTC')}}"
+          ng-minlength="0.00000001" ng-maxlength="10000000000" 
+          ng-model="amount" autocomplete="off" ng-disabled="sell.loading">
+          
+          <input ng-show="showAlternative" type="number" id="fiat" 
+          name="fiat" ng-attr-placeholder="{{'Amount in ' + (showAlternative ? 'USD' : 'BTC')}}" 
+          ng-model="fiat" autocomplete="off" ng-disabled="sell.loading">
+
+          <a ng-show="!showAlternative" class="postfix button" 
+            ng-click="showAlternative = true; amount = null">BTC</a>
+          <a ng-show="showAlternative" class="postfix button black" 
+            ng-click="showAlternative = false; fiat = null">USD</a>
+        </div>
+
+        <div class="text-center text-gray size-12 m10b">
+          <span ng-show="!(amount || fiat)">
+            Enter the amount to get the exchange rate
+          </span>
+          <span ng-show="!sell.sellPrice && (amount || fiat)">
+            Not available
+          </span>
+          <span class="label gray radius" ng-show="sell.sellPrice && amount && !fiat">
+            {{sell.sellPrice.amount * amount | currency : 'USD ' : 2}}
+          </span>
+        </div>
+
+        <div class="text-center">
+          <i class="db fi-arrow-down size-24 m10v"></i>
+        </div>
+
+        <label>Coinbase Account</label>
+        <div class="disabled-input">
+          {{index.coinbaseAccount.name}}
+        </div>
+
+        <div class="input m20t">
+          <input class="button black expand round" 
+                 ng-disabled="sell.loading || (!amount && !fiat)"
+                 ng-style="{'background-color': '#2b71b1'}"
+                 type="submit" value="Continue">
+        </div>
+      </form>
+    </div>
+  </div>
+
+  <div class="m20t row" ng-show="sell.sendInfo && !sell.sellInfo && !sell.success">
+    <div class="columns">
+      <h1>Funds sent to Coinbase Account</h1>
+      <p class="size-12 text-gray">
+        Bitcoin not confirmed yet. It will appear in activity as "Processing" transaction until it is confirmed. 
```

Comment:
The transaction is not yet confirmed. It will show as "Processing" in Activity. Your bitcoins will be sold automatically once it is confirmed.